### DISCUSSION TITLE
Worcester Bosch Comfort I RF thermostat supported?

### DISCUSSION THREAD

**Discussion by jthackray** [emsesp/EMS-ESP32#616] [General]
Hi,

I'm really impressed by your project. I've got a Worcester Bosch 30si Compact with a jack, and would like to set the temperature automatically by geo/time of day. It's controlled by a wireless Worcester Bosch Comfort I RF thermostat.

Can I write the desired temperature to the thermostat via the EMS bus using this hardware and software (e.g. from Home Assistant via MQTT) and if I set it to (say) 10 degrees Celsius, would this trigger the boiler to start heating the house? I can't see any mention of this thermostat in the docs, so I was wondering if it is supported or whether I'd need a new device.

Many thanks in advance,
Jonathan.

**Comment by MichaelDvP** [comment]
Sorry to say, but the RF thermostats we've seen before are not writable. See #138, #393, #448. 
But this thermostat is not kown, send a log which shows a change in setpoint on the RF thermostat, then we can see if there is a ems-telegram we can try to write to.
